packages/gatsby-theme-api-docs/src/components/type/properties/rows/name-type.js

```diff
@@ -26,6 +27,42 @@ const BetaWrapper = styled.span`
   padding-top: 0.2rem;
 `;
 
+function UnionParametersRow(props) {
+  const typesToRender = useTypesToRender({
+    property: props.types,
+    apiKey: props.apiKey,
+    isParameter: true,
+  });
+
+  return (
+    <>
+      Can be{' '}
+      {typesToRender.map(({ type }, idx, { length }) => {
+        return length > idx + 1 ? (
+          <span key={idx}>{type}, </span>
+        ) : (
+          <span key={idx}>or {type}</span>
+        );
+      })}
+    </>
+  );
+}
+
+UnionParametersRow.propTypes = {
```

Comment:
Style suggestion to update this to a tsx file instead of writing more old react propTypes

Reply:
We are currently still using js files for the api-docs components. We could definitely refactor the components to use typescript, but I would do it for the whole package and not just for a single file. Let's do this separately 👍🏼